tools/merkle_calculator/merkle_tree_calculator.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+  "fmt"
+  "time"
+  "flag"
+
+  "github.com/iotaledger/iota.go/trinary"
+  "github.com/iotaledger/iota.go/merkle"
+)
+
+// createMerkleTreeFile calculates a merkle tree and persists it into a file.
+func createMerkleTreeFile(filePath string, seed trinary.Hash, securityLvl int, depth int) error {
+
+  mt := merkle.CreateMerkleTree(seed, securityLvl, depth)
+
+  if err := merkle.StoreMerkleTreeFile(filePath, mt); err != nil {
+    return err
+  }
+
+  fmt.Printf("Merkle tree root: %v\n", mt.Root)
+
+  return nil
+}
+
+func main() {
+
+  var depth int
+  var securityLevel int
+  var seed string
+  var outputPath string
+
+  flag.IntVar(&depth, "depth", 0, "Depth of the Merkle tree to create")
+  flag.IntVar(&securityLevel, "securityLevel", 0, "Security level of the private key used")
+  flag.StringVar(&seed, "seed", "", "Seed for leaves derivation")
+  flag.StringVar(&outputPath, "output", "", "Output file path")
+
+  flag.Parse()
+
+  if depth < 1 {
+    fmt.Println("'depth' cannot be lower than 1")
+    return
+  }
+
+  if securityLevel < 1 || securityLevel > 3 {
```

Comment:
```suggestion
  if *securityLevel < 1 || *securityLevel > 3 {
```

Reply:
These vars are not pointers. Why do you attempt to dereference?